### PR TITLE
Rebrand to Trinsic w/ credentials, wallet, and provider nomenclature

### PR DESCRIPTION
**Pull request by drakebw**

### CODE REVIEW

File: csharp/TrinsicClientCredentials.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public StreetcredClientCredentials(IOptions<ServiceClientOptions> options)
         public override Task ProcessHttpRequestAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
             request.Headers.Add("Authorization", $"Bearer {Options.AccessToken}");
-            request.Headers.Add("X-Streetcred-Subscription-Key", Options.SubscriptionKey);
```
Comment (drakebw) [inline csharp/TrinsicClientCredentials.cs:33]: Don't change this yet

File: README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
-# Service Clients repository for Streetcred SSI platform
+# Service Clients repository for Trinsic SSI platform
 
-These service clients can be used in your web application with the Streetcred ID API.
+These service clients can be used in your web application with the Trinsic ID API.
 
-For more information and API Documentation, go to the [Streetcred ID API Documentation](https://docs.streetcred.id/docs/getting-started).
+For more information and API Documentation, go to the [Trinsic ID API Documentation](https://docs.trinsic.id/docs/getting-started).
```
Comment (drakebw) [inline README.md:5]: Update link

File: openapi/credentials.json [hunk 3]
```diff
@@ -11,8 +11,8 @@
             "email": "support@trinsic.id"
         }
     },
-    "host": "api.streetcred.id",
```
Comment (drakebw) [inline openapi/credentials.json:14]: Don't change this yet

File: README.md [hunk 4]
```diff
@@ -28,6 +28,6 @@ Coming soon...
 
 ## Samples
 
-Check the [samples repository](https://github.com/streetcred-id/samples) for client setup and usage
+Check the [samples repository](https://github.com/trinsic-id/samples) for client setup and usage
```
Comment (tmarkovski) [inline README.md]: Keep this to old URL. If we change the ORG name, github will automatically redirect old links to new one.

File: azure-pipelines.yml [hunk 5]
```diff
@@ -85,15 +85,15 @@ jobs:
   
   - task: CmdLine@2
     inputs:
-      script: 'autorest --input-file=openapi/credentials.json --typescript --output-folder=typescript --add-credentials --override-client-name=CredentialsServiceClient --source-code-folder-path=src/credentials --custom-service-client-options=[baseUri=''https://api.streetcred.id/'']'
+      script: 'autorest --input-file=openapi/credentials.json --typescript --output-folder=typescript --add-credentials --override-client-name=CredentialsServiceClient --source-code-folder-path=src/credentials --custom-service-client-options=[baseUri=''https://api.trinsic.id/'']'
```
Comment (tmarkovski) [inline azure-pipelines.yml]: Keep old URL

File: azure-pipelines.yml [hunk 6]
```diff
@@ -85,15 +85,15 @@ jobs:
...
 
   - task: CmdLine@2
     inputs:
-      script: 'autorest --input-file=openapi/wallet.json --typescript --output-folder=typescript --add-credentials --override-client-name=WalletServiceClient --source-code-folder-path=src/wallet --custom-service-client-options=[baseUri=''https://api.streetcred.id/'']'
```
Comment (tmarkovski) [inline azure-pipelines.yml]: Keep old URL

File: azure-pipelines.yml [hunk 7]
```diff
@@ -85,15 +85,15 @@ jobs:
...
+      script: 'autorest --input-file=openapi/wallet.json --typescript --output-folder=typescript --add-credentials --override-client-name=WalletServiceClient --source-code-folder-path=src/wallet --custom-service-client-options=[baseUri=''https://api.trinsic.id/'']'
      
   - task: CmdLine@2
     inputs:
-      script: 'autorest --input-file=openapi/provider.json --typescript --output-folder=typescript --add-credentials --override-client-name=ProviderServiceClient --source-code-folder-path=src/provider --custom-service-client-options=[baseUri=''https://api.streetcred.id/'']'
```
Comment (tmarkovski) [inline azure-pipelines.yml]: Keep old URL

File: openapi/credentials.json [hunk 8]
```diff
@@ -25,7 +25,7 @@
         },
         "subscriptionKey": {
             "type": "apiKey",
-            "name": "X-Streetcred-Subscription-Key",
```
Comment (tmarkovski) [inline openapi/credentials.json]: Keep existing key

File: openapi/provider.json [hunk 9]
```diff
@@ -11,7 +11,7 @@
         },
         "version": "v1"
     },
-    "host": "api.streetcred.id",
```
Comment (tmarkovski) [inline openapi/provider.json:14]: Keep existing host

File: typescript/src/trinsicClientCredentials.ts [hunk 10]
```diff
@@ -29,7 +29,7 @@ export class TrinsicClientCredentials implements ServiceClientCredentials {
 
     signRequest(webResource: WebResource): Promise<WebResource> {
         webResource.headers.set("Authorization", `Bearer ${this.accessToken}`);
-        webResource.headers.set("X-Streetcred-Subscription-Key", this.subscriptionKey);
```
Comment (tmarkovski) [inline typescript/src/trinsicClientCredentials.ts]: Keep existing

File: openapi/wallet.json [hunk 11]
```diff
@@ -25,7 +25,7 @@
         },
         "subscriptionKey": {
             "type": "apiKey",
-            "name": "X-Streetcred-Subscription-Key",
+            "name": "X-Trinsic-Subscription-Key",
```
Comment (tmarkovski) [inline openapi/wallet.json]: Keep existing

File: openapi/wallet.json [hunk 12]
```diff
@@ -11,7 +11,7 @@
         },
         "version": "v1"
     },
-    "host": "api.streetcred.id",
+    "host": "api.trinsic.id",
```
Comment (tmarkovski) [inline openapi/wallet.json]: Keep existing

**Pull request closed without merging**